data.qmd

```diff
@@ -292,6 +292,21 @@ df |>
   knitr::kable(digits = 3)
 ```
 
+### referral source
+
+```{r}
+#| label: referral_source-bar
+#| fig-cap: "referral_source"
```

Comment:
This is the text that appears by the caption and so would normally not be just the variable name `referral_source` but `"Referral Source"`.